docs/fsharp/language-reference/plaintext-formatting.md

```diff
@@ -83,14 +83,14 @@ where the type is interpreted as follows:
 | `%M` | a `System.Decimal` value  |    Formatted using the `"G"` format specifier for `System.Decimal.ToString(format)` |
 | `%O` | any value  |   Formatted by boxing the object and valling its `System.Object.ToString()` method |
```

Comment:
Is "valling" a real word?

Reply:
Probably, it's "calling" (the "c" and "v" keys are next to each other on the keyboard)

Reply:
@pkulikov Makes sense, I thought it might be some F# terminology I wasn't familiar with :).